knowage-vue/src/modules/managers/schedulationAgendaManagement/SchedulationAgenda.vue

```diff
@@ -259,4 +263,16 @@ export default defineComponent({
         }
     }
 }
+.custom-overflow-parent-container {
+    overflow: hidden;
+    min-width: 0;
+}
+.custom-overflow-auto {
+    min-width: 0;
+    overflow: auto;
+    flex-grow: 1;
+}
+.custom-kn-page-width {
+    width: calc(100vw - 58px);
```

Comment:
use calc(100vw - $mainmenu-width) instead. That parameter should be customizable.